packages/framework/ai-collab/src/explicit-strategy/index.ts

```diff
@@ -171,14 +167,54 @@ export async function generateTreeEdits(
 		throw error;
 	}
 
+	// Transform editLog into diffs
+	const diffs: Difference[] = editLog.map((log) => {
+		switch (log.edit.type) {
+			case "move": {
+				return {
+					type: "MOVE",
+					path: log.edit.destination as unknown as ObjectPath,
+					newIndex: log.edit.source as unknown as number,
+					value: log.edit.source as unknown as number,
+					objectId: log.edit.source as unknown as string,
+				};
+			}
+			case "insert": {
+				return {
+					type: "CREATE",
+					path: log.edit.destination as unknown as ObjectPath,
+					value: log.edit.content as unknown as TreeNode,
+				};
+			}
+			case "modify": {
+				return {
+					type: "CHANGE",
+					path: log.edit.target as unknown as ObjectPath,
+					value: log.edit.modification as unknown as TreeNode,
+					oldValue: log.edit.modification as unknown as TreeNode,
```

Comment:
This code has the `oldValue` set to the new value, not the old value. 

We will not be able to support `oldValue` fields here because we would need to capture the old value and turn it into JSON BEFORE its applied.

You can capture the old value do this by moving where you produce each `Difference`  down to the `generateTreeEdits()` function.